tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml

```diff
@@ -0,0 +1,39 @@
+[setting.USDe_max_collateral_limit]
+defaultValue = "<%= parseEther(String(50_000_000)) %>"
+
+[setting.USDe_issuance_ratio]
+defaultValue = "<%= parseEther('2.00') %>"
+
+[setting.USDe_liquidation_ratio]
+defaultValue = "<%= parseEther('1.05') %>"
+
+[setting.USDe_liquidation_reward]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.USDe_min_delegation]
+defaultValue = "<%= parseEther('100') %>"
+
+[invoke.CoreProxy_configureCollateral_USDe]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureCollateral"  # "args" see below in the multiline block
+
+[[invoke.CoreProxy_configureCollateral_USDe.args]]
+tokenAddress = "<%= settings.USDe_address %>"
+oracleNodeId = "<%= extras.USDe_oracle_id %>"
+issuanceRatioD18 = "<%= settings.USDe_issuance_ratio %>"
+liquidationRatioD18 = "<%= settings.USDe_liquidation_ratio %>"
+liquidationRewardD18 = "<%= settings.USDe_liquidation_reward %>"
+minDelegationD18 = "<%= settings.USDe_min_delegation %>"
+depositingEnabled = true
+
+[invoke.CoreProxy_setPoolCollateralConfiguration_USDe]
+target = ["system.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= extras.spartan_council_pool_id %>"]
+func = "setPoolCollateralConfiguration"
+args = [
+    "<%= extras.spartan_council_pool_id %>",
+    "<%= settings.USDe_address %>",
+    { collateralLimitD18 = "<%= settings.USDe_max_collateral_limit %>", issuanceRatioD18 = 0 },
```

Comment:
should issuanceRatioD18 be zero here?

Reply:
am pretty sure it is, but not sure why 😅

Reply:
yes. When it is zero for collateral config, then default global value is used.

Reply:
Should we have default value  or the sccp value?

Reply:
the "default" is a global value defined by `invoke.CoreProxy_configureCollateral_USDe` 
```
issuanceRatioD18 = "<%= settings.USDe_issuance_ratio %>"
```
on line 24 above. And it is as per SCCP - `200%`

This is a specific value for this collateral for pool 1, that does not need to override global value for this collateral.

Reply:
ohhh ok, i thought you meant by global, as in 1 value that rules them all... this is more of a local value, if it's specific to the USDe collateral i think 🤔